src/locales/en.json

```diff
@@ -648,7 +648,7 @@
       "header": "Areas",
       "subHeader": "Project areas",
       "area": "Area",
-      "instructions": "Several geometries can be drawn for the project. Geometry areas affect the project’s traffic congestion indices. Drawing tools can be found at the bottom left of the map view. With the pen tool, you can draw polygonal geometries by clicking the point where you want an angle. With the square tool, you can create square areas by first clicking the map and moving the mouse in the direction you want. Clicking it again “closes” the square drawing. A geometry can be edited by “grabbing” the edge of the geometry and dragging. A square geometry can also be shaped by grabbing the edge. lete a geometry, first select the geometry by clicking it; a bin icon will appear in the bottom left drawing tools which you can click to delete the geometry.",
```

Comment:
If I remember correctly, it's beneficial to leave the old translation in place. That way, the translator can use the old translation when doing the updated translation. No need to re-translate specific words etc.

Reply:
Good call! I reverted the change to have the old translation prefixed with the tag.